src/loader/index.js

```diff
@@ -0,0 +1,57 @@
+/* eslint no-empty:"off" */
+(function(w, d, globalVarName, appId) {
+    var initArgs;
+    var onCallArgs = [];
+    var next;
+    w[globalVarName] = {
+        init: function() {
+            initArgs = arguments;
+            return {
+                then: function(_next) {
+                    next = _next;
+                }
+            };
+        },
+        on: function() {
+            onCallArgs.push(arguments);
+        }
+    };
+
+    w.__onSmoochHostReady__ = function onHostReady(Lib) {
```

Comment:
This variable is hardcoded in the host lib so it can't really be configurable. I tried to find a way to make it dynamic but I didn't find any. The host lib needs something to notify when it's loaded.

I don't think it's too bad, under the hood a lot of stuff is still branded Smooch, but the public facing variable name is configurable.